app/controllers/items_controller.rb

```diff
@@ -19,7 +48,11 @@ def purl_preview
   def prepare
     if can_open_version? @object.pid
       begin
-        vers_md_upd_info = {:significance => params[:severity], :description => params[:description], :opening_user_name => current_user.to_s}
```

Comment:
Similar comment as before. Prefer not touching lines in a commit that addresses a bug, unless the lines touched relate to the issue at hand.